.github/workflows/tests-template.yml

```diff
@@ -16,7 +16,7 @@ permissions: read-all
 jobs:
   test-linux:
     # this is to prevent arm64 jobs from running at forked projects
-    if: ${{ github.repository == 'etcd-io/bbolt' || inputs.runs-on == 'ubuntu-latest' }}
+    if: ${{ github.repository == 'etcd-io/bbolt' || startsWith(inputs.runs-on, 'ubuntu') }}
```

Comment:
why you change this?

Reply:
> why you change this?

Sorry i didn't understand or misreadyour comment.

Reply:
It seems like you wrongly rebased/squashed your commits.

Reply:
Could you please guide me that how can I solve this?

Reply:
One way of solving this would be to first ensure that your branch is up to date:
```
git fetch origin
git rebase origin/main
```

Then, you could checkout from main, and amend your commit:

```
git checkout origin/main -- .github/workflows/tests-template.yml
git commit --amend
```

Reply:
Done all the steps as you said.